base/int.jl

```diff
@@ -412,13 +375,48 @@ if WORD_SIZE == 32
         (lolo&0xffffffffffffffff) + UInt128(w1)<<64
     end
 
-    div(x::Int128, y::Int128) = Int128(div(BigInt(x),BigInt(y)))
-    div(x::UInt128, y::UInt128) = UInt128(div(BigInt(x),BigInt(y)))
+    function div(x::Int128, y::Int128)
+        try
```

Comment:
Using try/catch here is going to be really slow. Would be better to use a test and branch for the one invalid case.